### DISCUSSION TITLE
User Groups or multiple separate instances on one server

### DISCUSSION THREAD

**Discussion by MarbuelB** [open-webui/open-webui#6795] [Ideas]
Hello,
I would like to ask about ideas how to be able to host Open WebUI for multiple groups of users. Ideally each group has one Admin and they do not see each others data/settings. It would be also nice if there was one "super admin" who can create admins of groups and enforce certain settings (like API keys/limits).
Alternatively, would it possible to have multiple instances of the WebUI running on a single server at different addresses? So that each group of users would simply use a different and specific URL to connect. Is that possible?
Thank you!
Best,
Marek

**Comment by flefevre** [comment]
Or why not have: everyone can create group and be the "owner" of a group and could declare "manager" able to add/delete user from the group
you can have multiple owner of a group.
it could be good is group is linked also with vector datase tenant to ensure the data is in silot and linked to the group.
